src/animations/timeLines.js

```diff
@@ -14,6 +13,21 @@ import {
 
 gsap.registerPlugin(MotionPathPlugin);
 
+export const moveHideItemAndChangeContentTl = (
+  itemRef,
+  changeContent,
+  newContentId
+) => {
+  const tl = gsap.timeline({ defaults: { ease: 'power4.out' } });
+  return tl
+    .to(itemRef, {
+      y: '+=220',
+      duration: 1,
+    })
+    .to(itemRef, { autoAlpha: 0, delay: 1 })
+    .eventCallback('onComplete', changeContent, [newContentId]);
```

Comment:
we could skip this and in previous `to` in the object, pass new property `onComplete`.